### DISCUSSION TITLE
[Feature] 必须是可以在网页签到的才可以使用吗？类似于夸克网盘签到可以吗

### DISCUSSION THREAD

**Discussion by yaoysyao** [qd-today/qd#443] [General]
### Verify steps

- [X] Tracker 我已经在 [Issue Tracker](……/) 中找过我要提出的问题
- [X] Need 当前 QD 框架并不包含该功能特性或者还不完善
- [X] Framework 这是 QD 框架应包含的特性，并非模板特性
- [X] Meaningful 我提交的不是无意义的 催促更新或修复 请求

### Describe the Feature

是否支持夸克网盘签到

### Describe the solution

夸克网盘好像只能在手机夸克浏览器签到，不能使用浏览器签到，这种该怎么处理，可以执行签到吗？

**Comment by a76yyyy** [comment]
只要你能抓到http请求就可以尝试制作模板，

此外这个issue不是针对框架的特性请求，因此我将其关闭并迁移到discussion里了
